hawkular-accounts-feature-pack-dist/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.hawkular.accounts</groupId>
+    <artifactId>hawkular-accounts-parent</artifactId>
+    <version>2.0.5.Final-SNAPSHOT</version>
+  </parent>
+  <artifactId>hawkular-accounts-feature-pack-dist</artifactId>
+  <name>Hawkular - Accounts - Feature Pack Distribution</name>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts-feature-pack</artifactId>
+      <version>${project.version}</version>
```

Comment:
I tried to stick with version literals my projects for some time knowing that `${project.version}` can be dangerous in some situations. However, I found out that version literals are not very practical when rebasing. They either cause conflicts or are not caught by the rebase thus staying on the outdated version.

The only problem I can remember being caused by `${project.version}` is that it does not work as expected when the pom is included. I think that this can hardly happen with non-BoM maven projects and I decided to rather bear that risc than loosing time on rebases/cherrypicks which already happened several times to me.

Anyway, I am ready to respect the conventions of the present git repo :)